acct/check-dns

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+"""Print DNS entries for domains associated with our users."""
+
+import sys
+
+from dns import resolver
+
+from ocflib.vhost import web
+from ocflib.vhost import mail
+from ocflib.vhost import application
+
+
+def main(username):
+
+    domains = set()
+
+    for k, v in web.get_vhosts().items():
+        if v['username'] == username:
+            domains.add(k)
+            domains.update(v['aliases'])
```

Comment:
what are k/v? can you call these `domain_name` and `vhost_config` or something? i don't actually know what they are